docs/builtin.rst

```diff
@@ -37,27 +41,72 @@ Data Grabbers
      - Type/Config
      - State
      - Version Added
-   * - `DataladHCP1200`
+   * - :class:`junifer.datagrabber.DataladHCP1200`
      - `HCP OpenAccess dataset <https://github.com/datalad-datasets/human-connectome-project-openaccess>`_
      - Open with registration
      - Built-in
      - In Progress
      - :gh:`4`
-   * - `JuselessDataladUKBVBM`
-     - UKB VBM dataset preprocessed with CAT. Available for Juseless only
+   * - :class:`junifer.configs.juseless.JuselessDataladUKBVBM`
+     - UKB VBM dataset preprocessed with CAT. Available for Juseless only.
      - Restricted
-     - `junifer.configs.juseless`
+     - ``junifer.configs.juseless``
+     - Done
+     - 0.0.1
+   * - :class:`junifer.datagrabber.DataladAOMIC1000`
+     - `AOMIC 1000 dataset <https://github.com/OpenNeuroDatasets/ds003097>`_
+     - Open without registration
+     - Built-in
      - Done
      - 0.0.1
+   * - :class:`junifer.configs.juseless.JuselessDataladAOMICVBM`
+     - AOMIC VBM dataset. Available for Juseless only.
+     - Restricted
+     - ``junifer.configs.juseless``
+     - In Progress
+     - :gh:`57`
+
 
+Planned
+-------
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - Name
+     - Description
+     - Access
+     - Type/Config
+     - Reference
+   * - CamCAN
+     - CamCAN dataset for Juseless
+     - Restricted
+     - ``junifer.configs.juseless``
+     - :gh:`46`
+   * - ENKI
+     - ENKI dataset for Juseless
+     - Restricted
+     - ``junifer.configs.juseless``
+     - :gh:`47`
+   * - IXI
+     - IXI dataset for Juseless
+     - Restricted
+     - ``junifer.configs.juseless``
+     - :gh:`48`
+   * - UCLA
+     - UCLA dataset for Juseless
+     - Open without registration
+     - ``junifer.configs.juseless``
```

Comment:
This should be it Built-in then, as there are no restrictions for the dataset.

Reply:
Got it, done.